docs/roman/references_general/distortion_selection.inc

```diff
@@ -0,0 +1,13 @@
+.. _distortion_selectors:
+
+Reference Selection Keyword Attributes for DISTORTION
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+CRDS selects appropriate DISTORTION references based on the following keyword attributes.
```

Comment:
DISTORTION references based
to 
DISTORTION reference files based
Q: can there be multiple distortion reference files?

Reply:
All these changes now added (modulo Nadia's requests).